src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -232,13 +232,23 @@ public override TaintedDataAbstractValue VisitObjectCreation(IObjectCreationOper
                     ProcessTaintedDataEnteringInvocationOrCreation(method, taintedArguments, originalOperation);
                 }
 
+                bool isSanitizingMethod = false;
                 PooledHashSet<string>? taintedTargets = null;
                 PooledHashSet<(string, string)>? taintedParameterPairs = null;
+                PooledHashSet<(string, string)>? sanitizedParameterPairs = null;
```

Comment:
This will need a `sanitizedParameterPairs?.Free()` in the `finally` block below.